### ISSUE TITLE
Improve tests when cross compiling

### ISSUE THREAD

**Issue by franziskuskiefer** [openmls/openmls#1574]
With #1573 we enable more tests for all provider implementations.
Unfortunately, when cross compiling the test proc macro picks up the supported ciphersuites of the host, not the target. Since proc macros are built for the host, there's no easy way around it. For now the tests of unsupported ciphersuites simple return without doing anything (thus pass).
This works but is not great for visibility in what's being tested. This issue tracks improving this.

**Comment by keks** [comment]
@franziskuskiefer do you think this blocks v0.6?

**Comment by franziskuskiefer** [comment]
I don't think this should block 0.6. But let's assess what exactly we want to do here and describe it. Or close it.
